### DISCUSSION TITLE
Server not found. Can you help me?

### DISCUSSION THREAD

**Discussion by pawatecnologia** [goastian/midori-desktop#89] [Q&A] [answered]
Guys, I'm having a problem that I'd like to share with you.

I don't know if it's a bug, so I didn't open one.

I installed the Midorian browser a long time ago and in the last few months it stopped updating and couldn't find the domain "repo.astian.org".

Have you guys ever experienced this? Do you know how I can solve it?

I even thought it was an IP block in my country, but I tested it with a remote browser and it gave the same situation.

![image](https://github.com/user-attachments/assets/64526701-c5f1-49af-8f69-922683497bfc)

remote browser: 
![image](https://github.com/user-attachments/assets/6acd1206-38a1-4e12-bfb4-8b9d98464e14)

**Comment by abejaranoh** [comment, marked as answer]
that repo is not working at the moment, but you can find the new packages under "Releases"
